operators/kas-fleetshard-bundle-template.yaml

```diff
@@ -0,0 +1,73 @@
+---
+apiVersion: template.openshift.io/v1
+kind: Template
+metadata:
+  name: kas-fleetshard
+objects:
+- apiVersion: operators.coreos.com/v1alpha1
+  kind: CatalogSource
+  metadata:
+    name: kas-fleetshard-catalog
+    namespace: ${NAMESPACE}
+    labels:
+      app: kas-fleetshard
+  spec:
+    sourceType: grpc
+    secrets:
+    - "rhoas-image-pull-secret"
+    image: ${BUNDLE_IMAGE}
+    displayName: Kas Fleetshard Catalog
+    publisher: Data Plane Team
+- apiVersion: operators.coreos.com/v1alpha1
+  kind: Subscription
+  metadata:
+    name: kas-fleetshard-subscription
+    namespace: ${NAMESPACE}
+    labels:
+      app.kubernetes.io/part-of: managed-kafka
+      app.kubernetes.io/component: kas-fleetshard-operator
+  spec:
+    channel: alpha
+    installPlanApproval: Automatic
+    name: kas-fleetshard-operator
+    source: kas-fleetshard-catalog
+    sourceNamespace: ${NAMESPACE}
+    config:
+      resources:
+        requests:
+          memory: "1Gi"
+          cpu: "500m"
+        limits:
+          memory: "1Gi"
+          cpu: "500m"
+      env:
+      - name: SSO_ENABLED
+        value: "true"
+      - name: MANAGEDKAFKA_ADMINSERVER_EDGE_TLS_ENABLED
+        value: "${MANAGEDKAFKA_ADMINSERVER_EDGE_TLS_ENABLED}"
+      - name: STANDARD_KAFKA_CONTAINER_CPU
+        value: 500m
+      - name: STANDARD_KAFKA_CONTAINER_MEMORY
+        value: 4Gi
+      - name: STANDARD_KAFKA_JVM_XMS
+        value: 1G
+      - name: STANDARD_ZOOKEEPER_CONTAINER_CPU
+        value: 500m
```

Comment:
@machi1990 - this is the YAML for the `Subscription` config.

Reply:
Perfect. I'll have a look tomorrow

Reply:
The idea that I've is to allow some of these to be configured directly via yaml file representing the content of the config. The file can be configured via a ConfigMap (KFM has tons of those already) - and it'll be loaded and applied directly. What do you think as an approach?

Reply:
The config yaml could look like:
```yaml
      resources:
        requests:
          memory: "1Gi"
          cpu: "500m"
        limits:
          memory: "1Gi"
          cpu: "500m"
      env:
      - name: SSO_ENABLED
        value: "true"
      - name: MANAGEDKAFKA_ADMINSERVER_EDGE_TLS_ENABLED
        value: "true"
      - name: STANDARD_KAFKA_CONTAINER_CPU
        value: 500m
      - name: STANDARD_KAFKA_CONTAINER_MEMORY
        value: 4Gi
      - name: STANDARD_KAFKA_JVM_XMS
        value: 1G
      - name: STANDARD_ZOOKEEPER_CONTAINER_CPU
        value: 500m
```

Reply:
+1 on that approach. It also aligns with how this is configured in the addon.

Reply:
I've opened https://github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/pull/971 to take care of this. Can you try it out?

Reply:
I'll try today, thanks.